manifest.json

```diff
@@ -112,6 +112,13 @@
           "js": ["app/angular.audio.js"]
         }
       },
+      "angularRecorder": {
+        "version": "1.4.1",
+        "downloadFormat": "zip",
+        "url": "https://github.com/logbon72/angular-recorder/archive/1.4.1.zip",
+        "rootDirPrefix": "angular-recorder-",
+        "targetDirPrefix": "angular-recorder-"
+      },
```

Comment:
Do you need more files from this zip other than *angular-audio-recorder.min.js*? If you don't, then try to way to download just *angular-audio-recorder.min.js* instead of the whole zip.

Reply:
Ah, ok, just to check how many files are in the unzipped folder?

Reply:
30 files (approx.)

Reply:
Ok, I think it is okay to keep it as zip for now.